test/evaluate.js

```diff
@@ -34,6 +35,7 @@ test('xdm (evaluate)', async function (t) {
 
   t.equal(
     renderToStaticMarkup(
+      // @ts-ignore runtime.js does not have a typing
       React.createElement((await evaluate('# hi!', runtime)).default)
```

Comment:
no clue how to do this, maybe better left for you @ChristianMurphy 😅

Reply:
Yeah, this will likely need to be handled upstream a definitely typed.
currently the runtime typing adds globals https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/jsx-runtime.d.ts but does not include exports.